lib/puppet/provider/reboot/windows.rb

```diff
@@ -2,8 +2,8 @@
 require 'open3'
 
 Puppet::Type.type(:reboot).provide :windows do
-  confine :operatingsystem => :windows
-  defaultfor :operatingsystem => :windows
+  confine :kernel => :windows
+  defaultfor :kernel => :windows
```

Comment:
Is that the standard way to confine Windows modules?  The core Facter facts seem to confine by kernel (c.f. [ipaddress.rb](https://github.com/puppetlabs/facter/blob/2.3.0/lib/facter/ipaddress.rb#L107), [virtual.rb](https://github.com/puppetlabs/facter/blob/2.3.0/lib/facter/virtual.rb#L190), and [system32.rb](https://github.com/puppetlabs/facter/blob/2.3.0/lib/facter/system32.rb#L13)).